apps/cookbook/src/app/examples/modal-example/modal-example-configuration/modal-example-configuration.component.ts

```diff
@@ -40,6 +40,12 @@ export class ModalExampleConfigurationComponent {
   @Input() openFullHeight: boolean;
   @Output() openFullHeightChange = new EventEmitter<boolean>();
 
+  @Input() interactWithBackground: boolean;
+  @Output() interactWithBackgroundChange = new EventEmitter<boolean>();
+
+  @Input() invertColors: boolean;
+  @Output() invertColorsChange = new EventEmitter<boolean>();
+
```

Comment:
As per tech refinement, this should be handled by exposing css custom props for background + text color and let the target project set those